docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/SymbolicStochasticGradientDescent.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Microsoft.ML.Samples.Dynamic
+{
+    public sealed class SymbolicStochasticGradientDescent
```

Comment:
>SymbolicStochasticGradientDescent [](start = 24, length = 33)

We also need to decide the fate of the classes in sample project. I see some of the classes are static some are sealed.